Orange/classification/logistic_regression.py

```diff
@@ -36,7 +36,7 @@ def coefficients(self):
 class LogisticRegressionLearner(SklLearner, _FeatureScorerMixin):
     __wraps__ = skl_linear_model.LogisticRegression
     __returns__ = LogisticRegressionClassifier
-    name = 'logreg'
+    name = 'logit'
```

Comment:
a) the number of results for "trump" is even greater, lets not call it that!
More seriously:
b) the number of results for "logistic regression" is over 5x that of "logit"
Even more seriously:
c) while I don't like nitpicking over terminology (especially when not very well defined), I think logit is more frequently used for the function, and logistic regression for the predictive model (which uses the function). Wikipedia has "logit regression" as an alternative, but I don't think it is used as much. And logit shows an article which has this at the top: "For the basic regression technique that uses the logit function, see logistic regression"

Anyway as long as we are not renaming the class to LogitRegression the name change gets a -1 from me. But, you could argue for a non abbreviated name if that bothers you...